### ISSUE TITLE
Work-related activity/Work-focused interview: orthography

### ISSUE THREAD

**Issue by stwalsh** [dwp/dwp-style-guide#27]
From Dean Smith on DWP Digital Slack:

>I'm seeing inconsistency in whether 'work-related activity' is hyphenated. Same for 'work-focused interview' (and whether there's 1 's' or 2 in focused). Would also help clarify that they don't need to be capitalised.

**Comment by JohnTurnbull99** [comment]
Yes, hyphenate compound adjectives. Don't need to be capitalised when written out. New style guide entry.

1 's' in focused. New style guide entry.

**Comment by lolylena** [comment]
Proposed text:

### Work-related activity [UC]
Lower case and hyphenated.


### Work-focused interview [UC]
Lower case and hyphenated. One 's' in ‘focused’.
